src/utils/daos_dump_errnos/daos_dump_errnos.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright 2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * Small utility to dump all descriptions of errcodes
+ */
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <cart/types.h>
+#include <cart/api.h>
```

Comment:
Could we just include daos_errno.h?

Reply:
> Could we just include daos_errno.h?

i ll give it a try

Reply:
daos_errno.h is sufficient, but since i moved it back to ftest, we end up linking with same libs as all other tests